binary/typedb

```diff
@@ -47,21 +47,34 @@ exit_if_java_not_found
 if [ -z "$1" ]; then
     echo "Missing argument. Possible commands are:"
     echo "  Server:          typedb server [--help]"
+    echo "  Node:            typedb node [--help]"
     echo "  Console:         typedb console [--help]"
     exit 1
 elif [ "$1" = "console" ]; then
     if [ -d "${TYPEDB_HOME}/console" ]; then
         SERVICE_LIB_CP="console/lib/*"
         CLASSPATH="${TYPEDB_HOME}/${SERVICE_LIB_CP}:${TYPEDB_HOME}/console/conf/"
     # exec replaces current shell process with java so no commands after this one will ever get executed
-        exec $JAVA_BIN ${CONSOLE_JAVAOPTS} -cp "${CLASSPATH}" -Dtypedb.dir="${TYPEDB_HOME}" com.vaticle.typedb.console.TypeDBConsole "${@:2}"
+        exec $JAVA_BIN ${JAVAOPTS} -cp "${CLASSPATH}" -Dtypedb.dir="${TYPEDB_HOME}" com.vaticle.typedb.console.TypeDBConsole "${@:2}"
     else
         echo "TypeDB Console is not included in this TypeDB distribution."
         echo "You may want to install TypeDB Console or TypeDB (all)."
         exit 1
     fi
-elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
-    if [[ -d "${TYPEDB_HOME}/server" ]]; then
+elif [[ "$1" = "server" ]] || [[ "$1" = "node" ]]; then
+    if [[ "$1" = "server" ]]; then
+        TYPEDB_SERVER_NAME="TypeDB (Server)"
+        TYPEDB_SERVER_MAIN=com.vaticle.typedb.core.server.TypeDBServer
```

Comment:
let's call this variable `TYPEDB_SERVER_CLASS`

Reply:
Done in 75b547695ddfb01b4d088a46e3bb578d9a1833cc